RELEASE_NOTES.md

```diff
@@ -82,3 +82,5 @@ CheckoutConfiguration(
 - When `CheckoutSessionProvider.createSession` to create a `CheckoutSession`, you can pass the `environment` and `clientKey` instead of the whole configuration.
 - In drop-in all actions will start in expanded mode
 - When using the Google Pay component, it is no longer necessary to manually import the `3ds2` module to handle transactions that require a native 3DS2 challenge. 
+- Removing stored payment methods are being handled internally if you are using Sessions integration. You do not need to override `onRemoveStoredPaymentMethod` function anymore.
+- Default api error message for drop-in changed from `Error sending payment. Please try again.` to `An unknown error occurred`.
```

Comment:
This can be any error that the merchant faced, we can't assume it's API. So maybe we can sat if you use `DropInServiceResult.Error` without specifying an error message, the default has changed?

Reply:
That is better indeed